astropy/wcs/__init__.py

```diff
@@ -1,21 +1,24 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
 .. _wcslib: http://www.atnf.csiro.au/~mcalabre/WCS/
-.. _Paper IV: http://www.atnf.csiro.au/people/mcalabre/WCS/index.html
+.. _WCS papers: http://www.atnf.csiro.au/people/mcalabre/WCS/index.html
+.. _Paper IV: http://www.atnf.csiro.au/people/mcalabre/WCS/dcs_20040422.pdf
 .. _SIP: http://irsa.ipac.caltech.edu/data/SPITZER/docs/files/spitzer/shupeADASS.pdf
-.. _ds9: http://hea-www.harvard.edu/RD/ds9/
-
-Introduction
-------------
 
 `astropy.wcs` contains utilities for managing World Coordinate System
 (WCS) transformations in FITS files.  These transformations map the
 pixel locations in an image to their real-world units, such as their
 position on the sky sphere.
 
-It is at its base a wrapper around Mark Calabretta's `wcslib`_, but
-also adds support for the Simple Imaging Polynomial (`SIP`_)
-convention and table lookup distortions as defined in WCS `Paper IV`_.
+It performs three separate classes of WCS transformations:
+
+- Core WCS, as defined in the `WCS papers`_, based on Mark
```

Comment:
May be use "FITS WCS standard" instead of "WCS papers" . The current paper IV (about time) is probably not in wcslib yet.